minecraft_servers/timenation/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "server_name": "timenation",
+  "nice_name": "TimeNation",
+  "direct_ip": "timenation.net",
+  "server_wildcards": [
+    "%.timenation.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://www.timenation.net",
+    "web_shop": "https://store.timenation.net",
```

Comment:
Website not available - Please fix or remove.

Reply:
Will be released shortly.